src/main/java/software/sigma/internship/dao/repo/StudentRepository.java

```diff
@@ -0,0 +1,7 @@
+package software.sigma.internship.dao.repo;
+
+import org.springframework.data.jpa.repository.JpaRepository;
+import software.sigma.internship.user.Student;
+
+public interface StudentRepository extends JpaRepository<Student, Long> {
```

Comment:
Please read about @Repository 
and write both StudentRepository and Teacher@Repository in one style